### DISCUSSION TITLE
A PR needs some love

### DISCUSSION THREAD

**Discussion by egao1980** [cffi/cffi#326] [Q&A] [answered]
Dear team, 

Is there a communication channel to make issues / PRs more visible to maintainers? 

We had PR for https://github.com/cffi/cffi/issues/321 for a while now and the original PR was even reviewed before so let's give it a little bit of a push :)

**Comment by sionescu** [comment, marked as answer]
There's the [mailing list](https://mailman.common-lisp.net/listinfo/cffi-devel) or the usual IRC channels (#lisp and #common-lisp on libera.chat).
